Kickstarter-iOS/DataSources/SettingsNotificationsDataSource.swift

```diff
@@ -8,7 +8,7 @@ public struct SettingsNotificationCellValue {
 }
 
 final class SettingsNotificationsDataSource: ValueCellDataSource {
-  weak var cellDelegate: SettingsNotificationCellDelegate?
+  weak var cellDelegate: SettingsNotificationsViewController?
```

Comment:
Why the type rename?

Reply:
`cellDelegate` is now the delegate for both `SettingsNotificationPickerCell` and `SettingsNotificationCell`. `SettingsNotificationsViewController` conforms to both of these delegate protocols.

Reply:
oh cool, seeing as conformance to the protocols is still the thing we care about here, can I suggest using Swift's protocol composition - either:

```swift
typealias SettingsNotificationCellDelegates = SettingsNotificationPickerCellDelegate & SettingsNotificationCellDelegate

weak var cellDelegate: SettingsNotificationCellDelegates?
```
or:

```swift
weak var cellDelegate: (SettingsNotificationPickerCellDelegate & SettingsNotificationCellDelegate)?
```